src/main/java/org/jenkinsci/test/acceptance/plugins/credentials/FileCredentials.java

```diff
@@ -9,7 +9,7 @@
 @Describable("Secret file")
 public class FileCredentials extends BaseStandardCredentials {
 
-    public Control file = control("file");
+    public Control file = control(by.name("file"));
```

Comment:
`control` searches by the `path` attribute.

In credentials 1.x, the `path` attribute was `file`. In credentials 2.x the `path` attribute is `definition/file`. Same for `secret`.

However, in both 1.x and 2.x, the name attribute is unchanged (for both secret and file)